lib/funky/video.rb

```diff
@@ -40,6 +40,11 @@ def picture
       data[:picture]
     end
 
+    # @return [String] the name of Facebook page for the video.
+    def page_name
+      data[:from] && data[:from][:name]
```

Comment:
You don't have to change that; my question was different.

The example you have in the test **does return** `from` and `name`.

But in your code you are suggesting some videos do not return `from` at all.

Is that true? Do you have an example of video that does not return `from`? 

If you do, add a test case for that